sdk/storage/Azure.Storage.Files.Shares/perf/Azure.Storage.File.Perf.slnf

```diff
@@ -0,0 +1,13 @@
+{
+  "solution": {
```

Comment:
nice.... I had no idea [this](https://docs.microsoft.com/visualstudio/ide/filtered-solutions?view=vs-2019#solution-filter-files) was a thing!

Reply:
Yes, I built and owned this feature while in the VS Project team. There are some related helpful things you can do when working with projects and solutions in the IDE to keep it responsive and the view clean.
There are other opportunities in this repo to use them and rely on a few 'uber' solution files.